observatorium/overlays/moc/smaug/thanos/deployments/observatorium-thanos-receive-controller.yaml

```diff
@@ -0,0 +1,65 @@
+kind: Deployment
```

Comment:
this already exists
https://github.com/operate-first/apps/blob/master/observatorium/base/instance/thanos/receive-controller/deployment.yaml